app.rb

```diff
@@ -1,2 +1,55 @@
 require 'sinatra'
-require 'sinatra/reloader'
\ No newline at end of file
+require 'sinatra/reloader'
+
+# # 99 Bottles of Beer
+#
+# ## Project Specs
+#
+# - The home page `/` should list:
+#   - "99 Bottles of beer on the wall"
+#   - add a link to take one down
+
+# counter = 100
+#
+# get '/' do
+#
+#   params[:num] = counter.to_i
+#   puts "hi"
+#   puts params[:num]
+#
+#   if counter > 0
+#     "bye"
+#     "#{params[:num]} Bottles of beer on the wall " + "<a href='/#{params[:num].to_i-1}'> Take One Down </a>"
+#     counter -= 1
+#   else
+#     "No bottles of beer on the wall"
+#   end
+#
+# end
+
+get '/:num' do
+
+  if params[:num].to_i>0
+  "#{params[:num]} Bottles of beer on the wall " + "<a href='/#{params[:num].to_i-1}'> Take One Down </a>"
+  else
+  "No bottles of beer on the wall"
+  end
+
+end
```

Comment:
👍 Nice! Be careful with your indentation, it makes a big difference in readability/maintainability